content/Hardware/Portenta Family/About-ADC-reference-voltage-on-Portenta-H7.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "About ADC reference voltage on Portenta H7"
```

Comment:
I think maybe we could mention in the title something like:

'How to use AREF pin on Portenta H7'

Reply:
I think the current title works because it mentions the end goal (ADC) as well as connecting to the specific pain point of not being able to set reference voltage with `analogReference()`. But when it's deployed I'll make sure that a search for 'How to use AREF pin on Portenta H7' returns this article, and make changes if needed.